cymetric/metrics.py

```diff
@@ -276,147 +276,28 @@ def transaction_quantity(series):
-    ('Transactions', ('ResourceId',), 'Commodity')
-    ]
-
-_swuschema = [('Year', ts.INT), ('SWU', ts.DOUBLE)]
-
-@metric(name='FcoSwu', depends=_swudeps, schema=_swuschema)
-def fco_swu(series):
-    """FcoSwu metric returns the separative work units required for each 
-    year in a 200-yr simulation. This is written for FCO databases that 
-    use the Bright-lite (i.e., the U235 and U238 are given separately 
-    in the FCO simulations).
-    """
-    tools.raise_no_pyne('SWU Required could not be computed', HAVE_PYNE)
-    mass = pd.merge(series[0].reset_index(), series[1].reset_index(),
-            on=['ResourceId'], how='inner').set_index(['ObjId', 'TimeCreated', 'NucId'])
-    swu = []
-    prods = {}
-    mass235 = {}
-    m = mass[mass['Commodity'] == 'LWR Fuel']
-    for (obj, _, nuc), value in m.iterrows():
-        if 922320000 <= nuc <= 922390000:
-            prods[obj] = prods.get(obj, 0.0) + value['Mass']
-        if nuc == 922350000:
-            mass235[obj] = value['Mass']
-    x_feed = 0.0072
-    x_tails = 0.0025
-    for obj, m235 in mass235.items():
-        x_prod = m235 / prods[obj]
-        swu0 = enr.swu(x_feed, x_prod, x_tails, product=prods[obj]) / 1e6
-        swu.append(swu0)
-    m = m.groupby(level=['ObjId', 'TimeCreated'])['Mass'].sum()
-    m = m.reset_index()
-    # sum by years (12 time steps)
-    swu = pd.DataFrame(data={'Year': m.TimeCreated.apply(lambda x: x//12),
-                             'SWU': swu}, columns=['Year', 'SWU'])
-    swu = swu.groupby('Year').sum()
-    rtn = swu.reset_index()
-    return rtn
-
-del _swudeps, _swuschema
-
-# Electricity Generated [GWe-y]
-_egdeps = [('TimeSeriesPower', ('Time',), 'Value'),]
-
-_egschema = [('Year', ts.INT), ('Power', ts.DOUBLE)]
-
-@metric(name='FcoElectricityGenerated', depends=_egdeps, schema=_egschema)
-def fco_electricity_generated(series):
-    """FcoElectricityGenerated metric returns the electricity generated in GWe-y 
-    in a 200-yr simulation. This is written for the purpose of FCO databases.
+@metric(name='ElectricityGenerated', depends=_egdeps, schema=_egschema)
+def electricity_generated(series):
+    """ElectricityGenerated metric returns the total electricity generated in 
+    MWe-y for each agent.
     """
     elec = series[0].reset_index()
-    # sum by years (12 time steps)
-    elec = pd.DataFrame(data={'Year': elec.Time.apply(lambda x: x//12), 
-                              'Power': elec.Value.apply(lambda x: x/1000)}, 
-                        columns=['Year', 'Power'])
-    elec = elec.groupby('Year').sum()
+    elec = pd.DataFrame(data={'SimId': elec.SimId,
+                              'AgentId': elec.AgentId,
+                              'Year': elec.Time.apply(lambda x: x//12), 
+                              'Power': elec.Value}, columns=['SimId', 'AgentId', 'Year', 'Power'])
+    el_index = ['SimId', 'AgentId', 'Year']
+    elec = elec.groupby(el_index).sum()
```

Comment:
I think this needs to divide by 12 to get MWe-y.  The current math gives MWe-months.